Landing Page

ark2002 opened the pull request:
The landing page with a component page which contains the dummy components list.

ark2002 committed:
feat: added landing page & component page

vaibhav18matere commented:
good work. One suggestion : the sub-heading is not clearly visible. you can increase it's `font-weight`

ark2002 committed:
fix: added responsiveness and updated the sub-heading

- Added code for the responsiveness of the page title for mobile users
- Increased the font-weight for the sub-headings for better visibility

ark2002 commented:
> good work. One suggestion : the sub-heading is not clearly visible. you can increase it's `font-weight`

hey ! fixed the sub-heading , you can check it out now !

vaibhav18matere commented:
> > good work. One suggestion : the sub-heading is not clearly visible. you can increase it's `font-weight`
> 
> hey ! fixed the sub-heading , you can check it out now !

awesome, good to go !

horsemaker commented:
![image](https://user-images.githubusercontent.com/60985700/152557692-813b5ccb-bf78-424a-b8d7-cf308a189e56.png)
there's no space at the bottom here, could we add some space so that it looks clean?

horsemaker commented on components_page.html:
```diff
@@ -0,0 +1,97 @@
+            <h2 class="header__logo">Astro UI</h2>
+          </div>
+        </a>
+        <nav class="header_nav">
```
Instead of  `header_nav`, shouldn't it be `header__nav`?

horsemaker commented on components_page.html:
```diff
@@ -0,0 +1,97 @@
+          </ul>
+        </nav>
+      </header>
+      <main class="main-content">
```
You could try to maintain a consistent naming convention. Shouldn't it be `main__content`?

ark2002 replied:
> You could try to maintain a consistent naming convention. Shouldn't it be `main__content`?

hey! Did the suggested changes , you can check it out now.

horsemaker replied:
Great! 👌

horsemaker commented on css/components_page.css:
```diff
@@ -0,0 +1,122 @@
+  justify-content: center;
+  gap: 2rem;
+}
+.intro_text {
```
`intro__text` here?

ark2002 committed:
fix: fixed the classnames & added space at the bottom

- fixed the class names to follow the proper naming convention
- Added a padding to the components__container class to avoid the buttons from touching the page-ends
